source/runtime_gui.cpp

```diff
@@ -1095,12 +1106,12 @@ void reshade::runtime::draw_gui()
 		{
 			for (const addon_info::overlay_callback &widget : info.overlay_callbacks)
 			{
-				if (widget.title == "OSD" ? show_splash || (!show_stats_window && !_show_overlay) : !_show_overlay)
-					continue;
-
-				if (ImGui::Begin(widget.title.c_str(), nullptr, ImGuiWindowFlags_NoFocusOnAppearing))
-					widget.callback(this);
-				ImGui::End();
+				if (!show_splash && (widget.title == "OSD" ? show_addons_osd : _show_overlay))
```

Comment:
Normal overlays can show while the splash message is displayed (like the main ReShade overlay), only the OSD one cannot, because they would overlap. That's why `show_splash` was only in the OSD condition previously.

Reply:
I tried only under reloading the effects.